bedrock/firefox/templates/firefox/browsers/mobile/index.html

```diff
@@ -142,19 +142,6 @@ <h3 class="mzp-c-menu-list-title">{{ ftl('browsers-mobile-download') }}</h3>
     </div>
 
   </section>
-  <section class="c-browser-compare mzp-l-content mzp-t-content-lg">
-    {% call split(
-      image=resp_img('img/firefox/browsers/dev.png',
-        srcset={ 'img/firefox/browsers/dev-high-res.png': '2x' },
-        optional_attributes={ 'class': 'mzp-c-split-media-asset'}
-      ),
-      block_class='t-compare'
-    ) %}
-      <h2>{{ ftl('browsers-mobile-see-how-firefox-for-desktop-compare') }}</h2>
-      <p class="c-compare-title mzp-has-zap-06">{{ ftl('browsers-mobile-see-how-firefox-for-desktop-strong') }}</p>
-      <p><a class="mzp-c-cta-link" href="{{ url('firefox.browsers.compare.index') }}" data-cta-type="link" data-cta-text="Compare">{{ ftl('browsers-mobile-compare') }}</a></p>
```

Comment:
Q: should the fluent strings stay defined for now?
https://github.com/mozilla/bedrock/blob/1519e9d9b3824b53461fde3baa2ca2677df8f746/l10n/en/firefox/browsers/mobile/index.ftl#L25-L29

Reply:
The comparison pages are being rewritten and it's possible we'll be able to reuse some of these strings. We can probably leave them defined-but-unused for now until we've figured out the next steps.

Reply:
That's what I thought. Only double checking;) 🥂